x-pack/plugin/sql/qa/src/main/resources/command.csv-spec

```diff
@@ -188,49 +188,49 @@ TODAY          |SCALAR
 showTables
 SHOW TABLES;
 
-       name        |     type      
-logs               |BASE TABLE
-test_alias         |VIEW          
-test_alias_emp     |VIEW          
-test_emp           |BASE TABLE     
-test_emp_copy      |BASE TABLE     
+       name        |     type   |  native   
```

Comment:
Still thinking whether the third column should be `native` or something else (`storage` doesn't fit either).

Reply:
I like `native` but how about `native_type` ?

Reply:
somewhat redundant - type, native type - plus should it be `native_type` or `native type`? A single word is much preferred.

Reply:
Imho the name of a previous column (`type`) is completely irrelevant of the next one, and I believe it's more clear to have it as `native_type`. And I would for sure vote for the `_` instead of a space: ` `.

Reply:
I opted for `kind` - one word, small (4 letters like the rest) and fairly accurate.